### ISSUE TITLE
A few things

### ISSUE THREAD

**Issue by Ezoteric** [RobertSkalko/Age-of-Exile#136]
First all great mod, really love the changes.
So there are a few things i would like to share. (my feedback)
**_Now some things may be incomplete cause you haven't implanted it yet and still working on it_** 
but i cant find any todo list so i will just point everything out i came across.

First of all, about the tiers, could u use 1 color range for all tier stuff, this incluses weapons, gear, gems, food, mats etc.
It can be confusing when one is used to a color range and then for other stuff the color range changes.
Gear = grey - green - yellow - purple - orange 
Mats. =  blue - yellow - green - orange - purple orsomething.
I am making a sphax texture pack so i can do this for my own pack already BUT i then also need to change the names of all items to the same color which i think i cant do for all the mats cause they dont have the font color in the file. or doesnt have a file to begin with.
So if you could use the "grey - green - yellow - purple - orange" order for all the other stuff that would be great and keel the blue for magic items which makes sense cause mana is blue :)

2nd: the magic tier
Talking about magic items, i have a sword, thats magical but only has physical dmg and no other stats, so what makes it magical ? same goes for any low lvl item that only has 1 stat that has nothing to do with magic.

3rd: textures
bunch of unused textures, are they yet to be deleted or planned on using ?

4th: added mods
World of exile config disable mod doesnt disable it.
Divine quests are nice, but i would like to be able to edit those, for example i would like to remove the loot chest quest cause looting 7 chests is not worth it cause in my modpack most chests have better loot then the quest will give.
I could not find any way to change any of this, i could only edit xp and dungeon key requirements.

5th: Dungeons
Be able to add more creatures to the dungeons list like the tower spawners would be nice.
Would also be nice to have a more spacious dungeon like youngs better stronghold (tbh that to me is really made for this mod).

Dont take any of these as negative feedback, but i just want to be able to make the most out of this mod to make the perfect mmo modpack possible for minecraft.
If anyhing i am willing to help out anyway possible tho i cant code :P

**Comment by RobertSkalko** [comment]
Magic items with no stat probably had a socket. It counts as a stat.

Divine missions are fully datapacks and 100% editable.
Same with dungeon mob lists. (in the dungeon dimension. Not the overworld ones)

**Comment by Ezoteric** [comment]
**_Divine missions are fully datapacks and 100% editable._**
I could not find anything related to the loot chest quest so also not a way to delete it.
If you could give me a example file that would be much appreciated 
Same goes for the dungeon dimension mob list, cant find it anywhere so i would have no idea where to add it.
i mean the towers have a config file on their own and the dungeon dimension does not have a config file.


**_Magic items with no stat probably had a socket. It counts as a stat._**

![image](https://user-images.githubusercontent.com/14137626/132894191-416d9b9e-4f45-498b-bb54-0022959f1066.png)

nothing, just dmg.

**Comment by RobertSkalko** [comment]
Look into minecraft datapacks. Search the mmorpg and divine_missions namespaces in the mod jar's data folders.

That sword has a stat, 10% phys damage.

**Comment by Ezoteric** [comment]
Yeah but a rare weapon also has a stat.
So basicly a magic item isnt something magic specific ?

About the datapacks, do you understand what i mean ? cause as i said before there no file where it states "loot * chest" or "Bring item" there is no file that has the specific task in detail that you can edit. Only a few files that lets you edit the exp you get.
It has 3 dirs -> pool, reward and tasks and in none of these files do i see the specific tasks that you see on the quest it self.

I want to delete the requirements -> loot  * chests 
![task](https://user-images.githubusercontent.com/14137626/132900128-4c25c17c-9c60-45e1-8779-28fa87d72b74.png)

**Comment by RobertSkalko** [comment]
data\divine_missions\task_entry\loot_a_chest.json
data\divine_missions\pool\general_aoe_tasks.json

**Comment by Ezoteric** [comment]
See i knew i wasnt dumb, i do not have these files....

![task](https://user-images.githubusercontent.com/14137626/132904629-8d746f68-b605-474a-8fe7-cf661749f4d0.png)
![task2](https://user-images.githubusercontent.com/14137626/132904727-37d57765-dd10-4b04-bfbe-20fdcd901907.png)

**Comment by TestSerber** [comment]
its probably in divine missions it is a jar file in aoe i think?

**Comment by Ezoteric** [comment]
Holy shit i finally found it in "Age-Of-Exile-1.16.5-2.1.5.jar\META-INF\jars\Divine-Missions.jar\data\divine_missions\task_entry\"
Thnx for the help and patience :)

PS: i just had a sandstone dungeon and it was way better than the small stone ones :D so nvm on the bigger stuff haha

**Comment by Ezoteric** [comment]
OK so i have deleted the loot chest quests from that file within my datapack and now it has refreshed missions 2 times and i get this crash:

Description: Exception in server tick loop

java.lang.NullPointerException: Exception in server tick loop
	at com.robertx22.divine_missions.saving.TaskData.createNew(TaskData.java:25)
	at com.robertx22.divine_missions.mission_gen.MissionUtil.createRandom(MissionUtil.java:106)
	at com.robertx22.divine_missions.mission_gen.MissionUtil.createRandomMissionItem(MissionUtil.java:32)
	at com.robertx22.divine_missions.components.data.PlayerMissionData.generateNew(PlayerMissionData.java:43)
	at com.robertx22.divine_missions.components.data.PlayerMissionData.onTick(PlayerMissionData.java:32)
	at com.robertx22.divine_missions.main.EventRegister.lambda$null$0(EventRegister.java:27)
	at com.robertx22.divine_missions.main.EventRegister$$Lambda$10674/1550330483.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at com.robertx22.divine_missions.main.EventRegister.lambda$reg$1(EventRegister.java:26)
	at com.robertx22.divine_missions.main.EventRegister$$Lambda$3369/1383907521.onEndTick(Unknown Source)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerTickEvents.lambda$static$2(ServerTickEvents.java:62)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerTickEvents$$Lambda$2694/1099082682.onEndTick(Unknown Source)
	at net.minecraft.server.MinecraftServer.handler$caf001$onEndTick(MinecraftServer.java:7763)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:853)
	at net.minecraft.class_1132.method_3748(class_1132.java:118)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at net.minecraft.server.MinecraftServer$$Lambda$8683/890894168.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

**Comment by Ezoteric** [comment]
This could be only once for every player that had the old quests and the quests gets reset, as we are now online and quests have reset again and no crash.

**Comment by RobertSkalko** [comment]
This is not the place for getting help to modify the mod. The discord is the place for that.

**Comment by Ezoteric** [comment]
You should also remove vines from the dungeons, we had litterly 25 zombies in this spot and was easy killing
![image](https://user-images.githubusercontent.com/14137626/132949690-5964d864-38ee-46c6-be2a-8a26eada518a.png)

**Comment by Ezoteric** [comment]
![image](https://user-images.githubusercontent.com/14137626/132949739-d4f87949-efcb-4630-8c39-945790f08e8e.png)

**Comment by Ezoteric** [comment]
there i was just standing AFK and they just go in there by themself and dont even atk :S
